base/applications/msconfig/autochkpage.c

```diff
@@ -0,0 +1,247 @@
+/*
+ * PROJECT:     ReactOS Applications
+ * LICENSE:     LGPL - See COPYING in the top level directory
+ * PURPOSE:     Auto Check (autochk) page handler
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+
+/* INCLUDES ******************************************************************/
+
+#include "precomp.h"
+#include "resource.h"
+#include "autochkpage.h"
+
+#include <tchar.h>
+#include <windows.h>
+
+/* VARIABLES *******************************************************************/
+
+HWND hAutoChkPage;
```

Comment:
Everything that can be a local should be a local, so most/all of these should live inside AutoChkPageWndProc.